site/in-dev/pynessie.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Python based CLI"
+---
+
+# Python based CLI
```

Comment:
SHould be put a warning here saying that this tool is not under active development anymore, and does not support v2?

Reply:
yes, added